src/layout/Hero/HeroSection.svelte

```diff
@@ -9,8 +12,18 @@
 			Customize your mobile experience through ReVanced <br /> by applying patches to your applications.
 		</p>
 		<div class="hero-buttons">
-			<Button icon="download" type="filled" href="download">Download</Button>
-			<Button icon="docs" type="tonal" href="patches">View patches</Button>
+			<Button type="filled" href="download">
+				<svelte:fragment slot="icon">
+					<TrayArrowDown size="20px" />
+				</svelte:fragment>
+				Download
+			</Button>
+			<Button type="tonal" href="patches">
+				<svelte:fragment slot="icon">
+					<FileDocumentOutline size="20px" />
+				</svelte:fragment>
+				View patches
+			</Button>
```

Comment:
Why the fragments? can't the icons be included with the text?

Reply:
makes it more obvious what it is?

Reply:
One of the only benefits of this PR is improving the DX when it comes to using icons, because inlining SVGs takes up too much screen estate and importing them doesn't allow them to be colored dynamically (without weird hacks)

This isn't much better if it's going to take 3 lines to display the icon, why not make the icon a parameter to the Button component or just inline it with the text?